api-report/tree.api.md

```diff
@@ -1168,12 +1170,10 @@ export interface TaggedChange<TChangeset> {
 // @alpha
 export type ToDelta = (child: NodeChangeset) => Delta.Modify;
 
-// @alpha (undocumented)
+// @alpha
 export enum TransactionResult {
```

Comment:
The PR description needs updating to reflect these changes.

Reply:
done.